draft-ietf-openpgp-pqc.md

```diff
@@ -956,7 +960,10 @@ scheme is as follows:
  9. Compute `C := AESKeyWrap(KEK, sessionKey)` with AES-256 as per {{RFC3394}}
     that includes a 64 bit integrity check
 
- 10. Output `eccCipherText || mlkemCipherText || len(C) || C`
+ 10. Output the algorithm specific part of the PKESK as
+     `eccCipherText || mlkemCipherText (|| symAlgId) || len(C) || C`, where
+     both `symAlgId` and `len(C)` are single octet fields and `symAlgId`
+     denotes the symmetric algorithm ID used and is present only for a v3 PKESK
```

Comment:
```suggestion
     denotes the symmetric algorithm ID used and is present only for a v3 PKESK.
```

Reply:
There are no periods after the preceding sentences in the algorithm specification. If we insert it here, we should do so in all algorithms in all steps.

Reply:
You are right, I had seen that somewhere else we have a period on the last one, but we should avoid here